utilities/cloudharness_utilities/deployment-configuration/helm/templates/auto-services.yaml

```diff
@@ -3,26 +3,25 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .app.name | quote }}
+  name: {{ .app.harness.service.name | quote }}
   labels:
-    app: {{ .app.name | quote }}
+    app: {{ .app.harness.deployment.name | quote }}
 {{ include "deploy_utils.labels" .root | indent 4 }}
 spec:
   selector:
-    app: {{ .app.name | quote }}
+    app: {{ .app.deploymentname | quote }}
   ports:
-    - port: {{ .app.port }}
+    - port: {{ .app.harness.deployment.port }}
       name: http
 {{- end }}
 
 
 {{- range $app := .Values.apps }}
-  {{- if and (hasKey $app "port") ($app.autoservice | default true)  }}
+  {{- if and (hasKey $app "port") $app.autoservice  }}
 ---
-  {{ include "deploy_utils.service" (dict "root" $ "app" $app) }}
   {{- range $subapp := $app }}
   {{- if contains "map" (typeOf $subapp)  }}
-  {{- if and (hasKey $subapp "port") ($subapp.autoservice | default false) }}
+  {{- if and (hasKey $subapp "port") $subapp.autoservice }}
```

Comment:
same here

Reply:
Addressed. As a reminder  subapps have to be tested and possibly reimplemented (now 1 level, a recursive strategy would fit better) - see #26